.github/workflows/test.yaml

```
@@ -0,0 +1,43 @@
+    strategy:
+      matrix:
+        node-version: [20]
+
+    steps:
+      - name: (>) Git checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: (>) Set up Node
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)